windows/installer_jack.nsi

```diff
@@ -0,0 +1,444 @@
+; Jamulus NSIS Installer with Modern User Interface
+; Installer is build with support for JACK on Windows
```

Comment:
```suggestion
; Installer for Jamulus with support for JACK on Windows
```
Grammar seems to be strange otherwise (build vs. built --> avoid it ;-))